### PR TITLE
Add a brief footer with a link to the repo

### PR DESCRIPTION
**Pull request by schwartzadev**
Of course, we can add to this and include whatever y'all like -- let me know if you would like to include any specifics and I'd love to add them!

### CODE REVIEW

File: templates/main/base.html [hunk 1]
```diff
@@ -37,5 +37,8 @@
     {% endblock %}
 {% endif %}
 {% block content %}{% endblock %}
+<footer>
+    <p>Developed collaboratively on <a href="https://github.com/discord-python/site">Github</a> • 2018</p>
```
Comment (gdude2002) [inline templates/main/base.html]: It's generally good practise to use HTML entities with unusual characters. Helps avoid possible encoding issues.

**Comment by lemonsaurus** [comment]
Does the UIKit add any special visual elements to the page based on this code? If so, can we see a picture in the PR of how it looks?

**Comment by lemonsaurus** [comment]
@schwartzadev did you see this comment?

**Comment by schwartzadev** [comment]
@1mn yes - I'm having trouble with my environment as of now -- maybe someone else could pull and take a screenshot?

**Comment by gdude2002** [comment]
Here's how it looks. Needs styling for sure.

image

**Comment by schwartzadev** [comment]
Here's how it looks now
sample

**Comment by lemonsaurus** [comment]
That's not too shabby, actually.

File: templates/main/base.html [hunk 2]
```diff
@@ -37,5 +37,12 @@
     {% endblock %}
 {% endif %}
 {% block content %}{% endblock %}
+<footer>
+<div class="uk-section uk-section-secondary uk-container-medium uk-text-meta">
+    <div class="uk-text-center">
+        <p>Developed collaboratively on <a href="https://github.com/discord-python/site">Github</a> • 2018</p>
```
Comment (lemonsaurus) [inline templates/main/base.html]: The • symbol should not be a literal, but instead should use the equivalent html entity.

**Review by lemonsaurus — CHANGES_REQUESTED** [review]
local_vars.env is not relevant to this PR.

**Review by lemonsaurus — APPROVED** [review]

**Comment by lemonsaurus** [comment]
This seems okay to me. We can merge if someone else approves of it as well. @gdude2002, can you look over it?

**Review by gdude2002 — APPROVED** [review]
Looking good! I would have liked it to stick to the bottom of the screen when the page is too short to fill it, but honestly that isn't a massive big deal, and it's *super complicated* (I just gave it a try myself).

Good work!

**Pull request merged**